src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery extends Ball {
+    private static final int MAX_BALL_VALUE = new Random().nextInt(100);
+    private final ColorSupplier colorSupplier = new ColorSupplier();
+
+    public Ball getRandomBall() {
+        Lottery newBall = new Lottery();
+        newBall.setColor(colorSupplier.getRandomColor().toString());
+        newBall.setNumber(MAX_BALL_VALUE);
```

Comment:
при такій реалізації в тебе Лотерея завжди буде одне і те саме число генерувати
```suggestion
        int ballNumber = random.nextInt(MAX_BALL_VALUE);
        newBall.setNumber(ballNumber);
```